src/createServer.js

```diff
@@ -1,3 +1,56 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const acceptableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => (
+  http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const originalText = req.url.split('?')[0].slice(1);
+    const params = new URLSearchParams(req.url.split('?')[1]);
+    const targetCase = params.get('toCase');
```

Comment:
You can use `new URL` and then parse everything that you need from that URL.

```suggestion
    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
   
    const originalText = normalizedURL.pathname.slice(1);
    const targetCase = normalizedURL.searchParams.get('toCase');
```